### PR TITLE
Added a decorator for registering a notification.

### PR DESCRIPTION
**Pull request by Jonothompson**

### CODE REVIEW

**Comment by jproffitt** [comment]
Thanks @Jonothompson! this looks great. The only feedback I have is that I think it would be better to put this inside the `NotificationRegistry` class. Just so it's all together. you could name the method `register_decorator` or something.

File: README.md [hunk 1]
```diff
@@ -45,10 +45,10 @@ Django library for separating the message content from transmission method
         
         Alternatively, a class decorator can be used.
         
+        from herald import registry
         from herald.base import EmailNotification
-        from herald.decorators import register_notification
         
-        @register_notification()
+        @register.register_decorator()
```
Comment (jproffitt) [inline README.md]: shouldn't this be `registry`?

File: herald/__init__.py [hunk 2]
```diff
@@ -19,6 +19,11 @@ def register(self, kls):
         Register a notificaion class
         """
 
+        from herald.base import NotificationBase
```
Comment (jproffitt) [inline herald/__init__.py]: can you change this to `from .base import NotificationBase`?

Comment (Jonothompson) [inline herald/__init__.py]: good catch! yes.

**Comment by jproffitt** [comment]
@Jonothompson looks good. Thanks!

**Comment by Jonothompson** [comment]
No problem, that was fun to figure out and learn.

**Pull request merged**